go.mod

```diff
@@ -1,14 +1,14 @@
 module sigs.k8s.io/external-dns
 
-go 1.14
+go 1.15
 
 require (
 	cloud.google.com/go v0.50.0
 	git.blindage.org/21h/hcloud-dns v0.0.0-20200807003420-f768ffe03f8d
 	github.com/Azure/azure-sdk-for-go v45.1.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.11.4
-	github.com/Azure/go-autorest/autorest/adal v0.9.2
-	github.com/Azure/go-autorest/autorest/azure/auth v0.5.1
+	github.com/Azure/go-autorest/autorest v0.11.10
```

Comment:
I know it's only patch updates, but has this been tested that external-dns still works on azure with the latest dep updates?

Reply:
If all good, we can merge it 🙂

Reply:
There's no breaking changes from what I can see, I can build and run it against a cluster too but not sure what's the testing procedure for patch version bumps in this repo